types/shares.go

```diff
@@ -56,35 +56,107 @@ func (m Message) MarshalDelimited() ([]byte, error) {
 	return append(lenBuf[:n], m.Data...), nil
 }
 
-func appendToShares(shares []NamespacedShare, nid namespace.ID, rawData []byte, shareSize int) []NamespacedShare {
-	if len(rawData) < shareSize {
-		rawShare := rawData
-		paddedShare := zeroPadIfNecessary(rawShare, shareSize)
+// appendToSharesContiguous appends one raw data separately as shares
+// Used for messages
```

Comment:
Comment is bit outdated and is not a full sentence.

Reply:
Fixed in 92301ea.